client/src/API.ts

```diff
@@ -22,20 +25,19 @@ export const startGame = async ({
   }
 
   const data = await response.json();
-  const command = castCommand(data.response);
-  const styles = extractTextStyles(command);
+  console.log(data);
```

Comment:
Should this be a `.debug`? Push back if not.

Reply:
Well, it does matter because if this gets called all the time it’s gonna clog the console. How often is this useful?